docs/podman-pod-restart.1.md

```diff
@@ -0,0 +1,48 @@
+% podman-pod-restart "1"
+
+## NAME
+podman\-pod\-restart - Restart one or more pods
+
+## SYNOPSIS
+**podman pod restart** [*options*] *pod* ...
+
+## DESCRIPTION
+Restart containers in one or more pods. Containers will be stopped if running and then restarted.
+Stopped containers will only be started. You may use pod IDs or names as input.
+
```

Comment:
And also, can you add a brief statement about the behavior of restarting multiple pods should an error occur in one of them what happens...